### ISSUE TITLE
The chroma Lora model is trained, and the generated image is black, loss=nan, can you help me solve this problem?

### ISSUE THREAD

**Issue by sundasheng1987** [tdrussell/diffusion-pipe#291]
The chroma Lora model is trained, and the generated image is black, loss=nan, can you help me solve this problem?

**Comment by Jamzsam** [comment]
I am having the exact same problem

**Comment by tdrussell** [comment]
Can you share your config? I have trained Chroma loras on a few different Chroma versions and it always worked fine. And does the loss immediately become NaN on the first step, or does it take some time for that to happen?

**Comment by Jamzsam** [comment]
Hi. These are my config files for chroma. (Yes the loss immediatley becomes NaN on the first step.)

# Dataset config file.
output_dir = '/home/------/diffusion-pipe6/diffusion-pipe/AbbChromaOutputs'
dataset = '//home//------//diffusion-pipe6//diffusion-pipe//Chroma_dataset.toml'

# Training settings
epochs = 1000
micro_batch_size_per_gpu = 1
pipeline_stages = 1
gradient_accumulation_steps = 4
gradient_clipping = 1.0
warmup_steps = 30
#offload_model = true
#blocks_to_swap = 20
# eval settings
eval_every_n_epochs = 1
eval_before_first_step = true
eval_micro_batch_size_per_gpu = 1
eval_gradient_accumulation_steps = 1

# misc settings
save_every_n_epochs = 10
checkpoint_every_n_minutes = 30
activation_checkpointing = 'unsloth'
partition_method = 'parameters'
save_dtype = 'bfloat16'
caching_batch_size = 1
steps_per_print = 1
#video_clip_mode = 'single_beginning'
video_clip_mode = 'none'
#blocks_to_swap = 12

[model]
type = 'chroma'
diffusers_path = '//home//jamzsam//diffusion-pipe6//diffusion-pipe//models//Flux'
transformer_path = '//home//jamzsam//diffusion-pipe6//diffusion-pipe//models//chroma//chroma-unlocked-v38-detail-calibrated.safetensors'
dtype = 'float16'
# You can optionally load the transformer in fp8 when training LoRAs.
transformer_dtype = 'float8'
flux_shift = true

[adapter]
type = 'lora'
rank = 32
#dtype = 'bfloat16'


[optimizer]
type = 'adamw_optimi'
lr = 1e-5
betas = [0.9, 0.99]
weight_decay = 0.01
eps = 1e-8
#stabilize = false

----------------------------------------------------------------------------------------------

Chroma_dataset.toml:

# Resolution settings.
# Can adjust this to 1024 for image training, especially on 24gb cards.
resolutions = [[1024, 1024]]

# Aspect ratio bucketing settings
enable_ar_bucket = true
min_ar = 0.5
max_ar = 2.0
num_ar_buckets = 12

# Frame buckets (1 is for images)
frame_buckets = [1]


[[directory]]
# Set this to where your dataset is
path = '/home/------/diffusion-pipe6/diffusion-pipe/training_data/images/Abb-Chroma'
mask_path = '/home/------/diffusion-pipe6/diffusion-pipe/training_data/images/Abb-Chroma-Mask'
# Reduce as necessary
num_repeats = 1
